packages/styled-components/src/test/__snapshots__/ssr.test.js.snap

```diff
@@ -3,20 +3,20 @@
 exports[`ssr should add a nonce to the stylesheet if webpack nonce is detected in the global scope 1`] = `"<h1 class=\\"sc-b c\\">Hello SSR!</h1>"`;
 
 exports[`ssr should add a nonce to the stylesheet if webpack nonce is detected in the global scope 2`] = `
-"<style nonce=\\"foo\\" data-styled data-styled-version=\\"JEST_MOCK_VERSION\\">body{background:papayawhip;}
-data-styled.g1[id=\\"sc-global-a\\"]{content:\\"sc-global-a,\\"}
-.c{color:red;}
-data-styled.g2[id=\\"sc-b\\"]{content:\\"c,\\"}
+"<style nonce=\\"foo\\" data-styled data-styled-version=\\"JEST_MOCK_VERSION\\">.c{color:red;}
+data-styled.g1[id=\\"sc-b\\"]{content:\\"c,\\"}
+body{background:papayawhip;}
+data-styled.g2[id=\\"sc-global-a1\\"]{content:\\"sc-global-a1,\\"}
```

Comment:
this switches position in these snapshots because the component ID is registered lazily now for cGS since we can't know the instance # until render time